frontend/src/tests/instructor-permission-test.js

```diff
@@ -94,7 +244,17 @@ export function instructorsPermissionTests(api) {
         expect(resp).toHaveStatus("success");
     });
 
-    it.todo("can't update applications");
+    it("can't update applications", async () => {
+        await cantUpdateTestFrame(
+            `/instructor/sessions/${session.id}/applications`,
+            null,
+            `/instructor/applications`,
+            `/admin/applications`,
+            (application) => {
+                application.program = "test program";
+            }
+        );
```

Comment:
I don't like how most of the code was moved out of the tests. I cannot look at this code an understand what steps the test is actually testing.

Take an approach more like #487 with setup and teardown functions.